jenkins/jjb/project.yaml

```diff
@@ -0,0 +1,29 @@
+- project:
+    name: 'nightly-tests-trigger'
+    jobs:
+      - '{repo}-{version}-trigger':
+          repo: 'vic'
+          version: 'master'
+          branch: 'master'
+      - '{repo}-{version}-trigger':
+          repo: 'vic'
+          version: '1.4.3'
+          branch: 'releases/1.4.3'
+
+- project:
+    name: 'nightly-tests'
+    default_esx_build: '8169922'
+    default_vc_build: '8217866'
+    build_timeout: '7200'
+    parallel_jobs: '4'
+    jobs:
+      - '{repo}-{version}-nightly':
+          repo: 'vic'
+          version: '1.4.3'
+          branch: 'releases/1.4.3'
+          addition_env: 'VCH_BRANCH=releases/1.4.3'
```

Comment:
What's the purpose of this?

Reply:
To generate similar jobs from the same template.       addition_env: 'VCH_BRANCH=releases/1.4.3' is a workaround because I can't set VCH_BRANCH=master.

Reply:
There is no master folder and build are stored directly under bucket if it is from master branch

Reply:
I see. Makes sense!